specification/openapi.json

```
@@ -0,0 +1,261 @@
+          },
+          "status": {
+            "type": "string",
+            "enum": ["active", "disabled"],
+            "description": "Should the coin be processed"
+          },
+          "createCoin": {
+            "type": "boolean",
+            "description": "Should an app itself create the coin or only use the info for the blockchain"
+          },
+          "defaultVisibility": {
+            "type": "boolean",
+            "description": "To show a coin by default, or hide it"
+          },
+          "defaultOrdinalLevel": {
+            "type": "integer",
+            "description": "Default ordinal number in a wallet list. Coins with the same ordinal number are sorted alphabetically. Coins without an order are shown last, alphabetically"
+          },
+          "consensus": {
+            "type": "string",
+            "description": "Blockchain consensus type"
+          },
+          "blockTimeFixed": {
+            "type": "integer",
+            "description": "Fixed block time in ms"
+          },
+          "blockTimeAvg": {
+            "type": "integer",
+            "description": "Average block time in ms"
+          },
+          "txFetchInfo": {
+            "type": "object",
+            "properties": {
+              "newPendingInterval": {
+                "type": "integer",
+                "description": "Interval between fetching Tx in ms when its current status is \"Pending\" for new transactions"
+              },
+              "oldPendingInterval": {
+                "type": "integer",
+                "description": "Interval between fetching Tx in ms when its current status is \"Pending\" for old transactions"
+              },
+              "registeredInterval": {
+                "type": "integer",
+                "description": "Interval between fetching Tx in ms when its current status is \"Registered\""
+              },
+              "newPendingAttempts": {
+                "type": "integer",
+                "description": "Attempts to fetch Tx when its current status is \"Pending\" for new transactions"
+              },
+              "oldPendingAttempts": {
+                "type": "integer",
+                "description": "Attempts to fetch Tx when its current status is \"Pending\" for old transactions"
+              }
+            }
+          },
+          "txConsistencyMaxTime": {
+            "type": "integer",
+            "description": "Time in ms when difference between in-chat transfer and Tx timestamp considered as acceptable"
+          },
+          "nodes": {
+            "type": "array",
+            "items": {
+              "type": "object",
+              "properties": {
+                "url": {
+                  "type": "string"
+                }
+              }
+            },
+            "description": "Node links for API"
+          },
+          "links": {
+            "type": "array",
+            "items": {
+              "type": "object",
```

The array is optional but the object properties must be required. At least the `url` param.

```suggestion
              "type": "object",
              "required": ["name", "url"],
```